mcsema/BC/Function.cpp

```diff
@@ -428,6 +428,47 @@ static void LiftBlockIntoFunction(TranslationContext &ctx) {
   }
 }
 
+static void AllocStackVars(
+    llvm::BasicBlock *bb,
+    const NativeFunction *cfg_func){
+  auto func = bb->getParent();
+  auto word_type = llvm::Type::getInt64Ty(func->getContext());
```

Comment:
The global variable `gWordType` is an integer type that is pointer sized. That is what you probably want to use.